Updates to README.md (#13464)
desktop/desktop: README.md

# [GitHub Desktop](https://desktop.github.com)

[GitHub Desktop](https://desktop.github.com/) is an open source [Electron](https://www.electronjs.org/)-based

> Official GitHub app. It is written in [TypeScript](http://www.typescriptlang.org) and

Reviewer: Let's avoid the "official" part here. That should be evident by the name and I don't we don't need to be more direct than that.

```suggestion
GitHub app. It is written in [TypeScript](http://www.typescriptlang.org) and
```

You can install this alongside your existing GitHub Desktop for Mac or GitHub
Desktop for Windows application.

> For more information on Github Desktop please visit this [article](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop)

Reviewer: Since we already have a "More resources" section in the readme I think it would make sense to move this paragraph underneath that and to tweak it to match the other link already in that section

```suggestion
See our [getting started documentation](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop) for more information on how to set up, authenticate, and configure GitHub Desktop.
```

Could you make that change and then move it to the more resources section?

> See our [getting started documentation](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop) for more information on how to set up, authenticate, and configure GitHub Desktop.

Reviewer: Could we move this under the "More resources" header please?

Author: Yes, but I can't see a commit changes button, like I saw for your previous edit requests @niik

Reviewer: Right, that's because suggested edit's, as they're called, work best for same line suggestions. They're a great tool for me to make small suggestions but for larger changes or, like in this case, moving a line somewhere else, we still have to resort to manually making the change and committing. You can update this branch by clicking on the `Files changes` tab and then selecting edit file.

![image](https://user-images.githubusercontent.com/634063/145223675-3eca672a-df2b-409d-ae51-2c950124568e.png)

That will give you a web editor if you would prefer to not make the change locally.

Author: I can not see the More Resources header.

Reviewer: It's almost at the end of README.md (https://github.com/desktop/desktop/blob/05de6f4276c3b756271cdf68d3ce9847629a792d/README.md#more-resources)

Line 87

![image](https://user-images.githubusercontent.com/634063/145224420-8a62f3b0-165a-4718-a16f-804faf0809de.png)

Author: Done

If you're looking for something to work on, check out the [help wanted](https://github.com/desktop/desktop/issues?q=is%3Aissue+is%3Aopen+label%3A%22help%20wanted%22) label.

> # Purpose
> The purpose of Github Destop is that, it allows you to collaboarate with your team, using some of the best practices. 
> 

Reviewer: A purpose section like what you're suggesting is probably better suited on a marketing page like `desktop.github.com`. Since this README is primarily aimed for contributors I think we should keep it focused on that.

```suggestion
```

Author: True
